### DISCUSSION TITLE
Can I save the model offline? So it will not always try to download it from huggingface?

### DISCUSSION THREAD

**Discussion by aditya39** [SYSTRAN/faster-whisper#246] [Q&A] [answered]
A couple days ago I try run my program, but it give me bad gateway error.
After I check, it was because huggingface web cannot be access at that time, so the model failed to be download.

Is there's any way to save the model offline?

Thank you

**Comment by guillaumekln** [comment, marked as answer]
See #116 for a way to download the model.

FYI, the next version of the library will be more robust to temporary connection issues to the Hugging Face Hub: https://github.com/guillaumekln/faster-whisper/pull/215
